src/index.html

```diff
@@ -0,0 +1,133 @@
+                alt="">
+        </div>
+        <div id="search-area">
+            <form>
+                <input type="text" placeholder="Search">
+                <button id="search"><i class="fa fa-search"></i></button>
+            </form>
+            <button id="mic"><i class="fa fa-microphone" title="Search with your voice"></i></button>
+        </div>
+
+        <div id="right">
+            <i class="fa fa-video-camera" title="Create"></i>
+            <i class="fa fa-th" title="YouTube apps"></i>
+            <i class="fa fa-bell" title="Notifications"></i>
+            <span>M</span>
+        </div>
+    </nav>
+
+    <section>
```

Comment:
- [ ] Great work with the semantic HTML tags but I want to suggest a few things that can really improve the way you use them. I think if you changed this section tag to ```<main></main>``` and changed the child divs to ```<section></section>``` tags instead, your code would be semantically friendlier. Read more on semantic HTML [here](https://www.w3schools.com/html/html5_semantic_elements.asp)